build.gradle

```diff
@@ -221,19 +219,29 @@ def fatJavadocJar(name, projects) {
 }
 
 // --------------------------------------
-// StatsD fat jar
+// StatsD reporter fat jar
 // --------------------------------------
 
 def statsd_fat_jar_name = 'tally-statsd-fat'
 def statsd_fat_jar_projs = [':tally-core', ':tally-statsd']
 
+// --------------------------------------
+// M3 reporter fat jar
```

Comment:
If I read this right, you're only creating fat jars. Do you also want to create thin jars?

Reply:
The thin jars are created automatically, configured in the `subprojects` section of this build file. The fat jars need explicit logic to create, since the parent project is just a shell to contain the subprojects and does not have any code of its own.